### PR TITLE
Path cohort

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

*Please describe.*  
- expose button for creating cohort from paths
*If this affects the frontend, include screenshots.*  

## How did you test this code?

*Please describe.*
- manual QA creating path cohorts for total, dropoff, success values

### CODE REVIEW

File: frontend/src/scenes/trends/PersonsModal.tsx [hunk 1]
```diff
@@ -94,10 +94,7 @@ export function PersonsModal({
     )
 
     const isDownloadCsvAvailable: boolean = view === InsightType.TRENDS && showModalActions && !!people?.action
-    const isSaveAsCohortAvailable =
-        clickhouseFeaturesEnabled &&
-        (view === InsightType.TRENDS || view === InsightType.STICKINESS || view === InsightType.FUNNELS) &&
-        showModalActions
+    const isSaveAsCohortAvailable = clickhouseFeaturesEnabled && showModalActions
```
Comment (neilkakkar) [inline frontend/src/scenes/trends/PersonsModal.tsx:97]: what about lifecycle and retention? 👀

File: frontend/src/scenes/trends/personsModalLogic.tsx [hunk 2]
```diff
@@ -129,6 +129,7 @@ export const personsModalLogic = kea<personsModalLogicType<LoadPeopleFromUrlProp
         setSearchTerm: (term: string) => ({ term }),
         setCohortModalVisible: (visible: boolean) => ({ visible }),
         loadPeople: (peopleParams: PersonsModalParams) => ({ peopleParams }),
+        setUrl: (props: LoadPeopleFromUrlProps) => props,
```
Comment (neilkakkar) [inline frontend/src/scenes/trends/personsModalLogic.tsx:132]: always objects for actions plis!
```suggestion
        setUrl: (props: LoadPeopleFromUrlProps) => ({props}),
```

Comment (neilkakkar) [reply frontend/src/scenes/trends/personsModalLogic.tsx:132]: Huh, wait, this is weird, in the reducer below, it passes the props, not these props^. What?

Comment (EDsCODE) [reply frontend/src/scenes/trends/personsModalLogic.tsx]: done

File: frontend/src/scenes/trends/PersonsModal.tsx (hunk 1 above)
Comment (EDsCODE) [reply frontend/src/scenes/trends/PersonsModal.tsx]: Good call. Retention uses a different modal so this wont impact, but forgot about lifecycle. Also, those still need a tiny bit of API additions. I just readded the conditions

**Review by neilkakkar — APPROVED** [review]
Some type annotations seem borked, but otherwise LGTM!

(and if it's too much of a pain to fix those annotations, ok with going back to `prop` from `{ prop }` )

**Pull request merged**